lib/Doctrine/Adapter/Statement/Mock.php

```diff
@@ -69,7 +69,9 @@ public function __construct($mock)
      * @return boolean              Returns TRUE on success or FALSE on failure
      */
     public function bindColumn($column, $param, $type = null)
-    { }
+    {
+        return false;
```

Comment:
Preventing from having to change this to `void` return which would cause a different error with the mis-matched return types between parent and child.